idr0015-UNKNOWN-taraoceans/idr0015-study.txt

```diff
@@ -1,40 +1,50 @@
 "# Section with generic information about the study including title, description, publication details (if applicable) and contact details"					
 # Study					
 Comment[IDR Study Accession]	idr0015				
-Study Title	Plankton Environmental High-Content Fluorescence Microscopy (e-HCFM)				
+Study Title	Quantitative 3D-imaging for cell biology and ecology of environmental microbial eukaryotes				
 Study Type	high content screen				
 Study Type Term Source REF	EFO				
 Study Type Term Accession Number	EFO_0007550				
-Study Description	Images from sea water samples collected a different geographic locations and imaged using high-content fluorescence microscopy				
+Study Description	"We present a 3D-fluorescence imaging and classification tool for high throughput analysis of microbial eukaryotes in environmental samples. It entails high-content feature extraction that permits accurate automated taxonomic classification and quantitative data about organism ultrastructures and interactions. Using plankton samples from the Tara Oceans expeditions, we validate its applicability to taxonomic profiling and ecosystem analyses, and discuss its potential for future integration of eukaryotic cell biology into evolutionary and ecological studies. This study is subject to the data release policy described here: http://www.embl.de/tara-oceans/start/Tara-Oceans-Data-Release-Policy.pdf"				
 Study Organism	mixed sample				
-Study Organism Term Source REF					
-Study Organism Term Accession Number					
+Study Organism Term Source REF	NCBITaxon				
+Study Organism Term Accession Number	NCBITaxon_1427524				
 Study Screens Number	1				
 Study External URL					
 Study Public Release Date					
+Study Version History	"November 2017 - the study title, description, publication and screen name updated."				
 					
 # Study Publication					
-Study PubMed ID					
-Study Publication Title					
-Study Author List					
+Study PubMed ID	29087936				
+Study Publication Title	Quantitative 3D-imaging for cell biology and ecology of environmental microbial eukaryotes.				
+Study Author List	"Colin S, Coelho LP, Sunagawa S, Bowler C, Karsenti E, Bork P, Pepperkok R, de Vargas C"				
+Study PMC ID	PMC5663481				
+Study DOI	http://dx.doi.org/10.7554/eLife.26066				
+					
+# Study License and Data DOI					
+Study License	CC-BY 4.0				
+Study License URL	https://creativecommons.org/licenses/by/4.0/				
+Study Copyright	Colin et al				
+Study Data Publisher	University of Dundee				
+Study Data DOI	http://dx.doi.org/10.17867/10000108 				
 					
 # Study Contacts					
 Study Person Last Name	de Vargas				
 Study Person First Name	Colomban				
 Study Person Email	vargas@sb-roscoff.fr				
 Study Person Address	Station Biologique de Roscoff				
-Study Person Roles					
+Study Person Roles	Principal Investigator				
 					
-Term Source Name					
-Term Source File					
+Term Source Name	NCBITaxon	EFO			
```

Comment:
Missing tab?

Reply:
The tab is there, don't know why it looks like there isn't one in github

Reply:
You're right. GitHub diff issue